libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -448,13 +449,15 @@ class AP_InertialSensor : AP_AccelCal_Client
     bool _new_accel_data[INS_MAX_INSTANCES];
     bool _new_gyro_data[INS_MAX_INSTANCES];
 
-    // optional notch filter on gyro
+    // optional notch filter on gyro and accel
     NotchFilterParams _notch_filter;
     NotchFilterVector3f _gyro_notch_filter[INS_MAX_INSTANCES];
+    NotchFilterVector3f _accel_notch_filter[INS_MAX_INSTANCES];
```

Comment:
Is this worth it? I would simply only provide the harmonic notch for accels

Reply:
This PR started with the static notch on accel, I would keep this one, is just another option for the user, the code is simple and tested and has a backward compatible param.
But let me know if there's a particular reason to remove it